OperatingSystem-Services/Platform-Linux/OpenVAS/README.md

```diff
@@ -0,0 +1,419 @@
+	```
+- Downloads `docker-compose.yml` to the greenbone download directory.
+
+4. **Configure Docker Compose File**
+	- **Switch Image Version to Latest**
+		- Several containers had issues with running the default docker images. *(You may not need to do this if there are no issues with gvmd.)* To fix this you must change the the image version from *stable* to **latest**.
+			```
+			sudo nano $DOWNLOAD_DIR/docker-compose.yml
+			```
+			Navigate to the following containers: **pg-gvm**, **gvmd**, **gsa**, **openvas**, **openvasd**, **ospd-openvas**.
+			Each of these containers has an image option. Ex:
+			<img  src="Images/ContainerStable.png">
+			Change every image listed above from `:stable` to `:latest`. Ex:
+			<img  src="Images/ContainerLatest.png">
+			
+	- **Expose Greenbone Security Assistant**
+		- The `docker-compose.yml` file must be configured to expose the Greenbone Security Assistant (GSA) to an external interface. 
+			```
+			sudo nano $DOWNLOAD_DIR/docker-compose.yml
+			```
+			Navigate to the **gsa** image and change the ports from `127.0.0.1:9392:90` to either `0.0.0.0:9392:80` or `<EXTERNAL_IP>:9392:80`
+			<img  src="Images/gsaHTTPS.png">
+	- **Configure `https` instead of `http`**
+        >[!WARNING] DO NOT USE HTTP
+
+		- By default GSA will use `http` which is very dangerous unless you  want all of your network's vulnerabilities to be public.
+			To solve this we must reconfigure the **gsa** container once again.
+			We will configure **gsa** to use port `443` instead of `80` as well as specify a certificate and key.
+			Your **gsa** container should look like:
+            <img  src="Images/gsaSecrets.png">
+			
+			This provides the container with a certificate and key in the default locations.
+			
+		- You must additionally add the secrets to the bottom of `docker-compose.yml`.
+			<img  src="Images/Secrets.png">
+			This defines the secrets used in the gsa container.
+			
+		- The `server-certificate` and `private-key` can either be self signed certificates or can be obtained from a CA.
+			In this example I will use a self signed certificate. Generate a certificate and key at the location you chose for your secrets:
+			```
+			openssl req -x509 -newkey rsa:4096 -keyout /home/<USER>/docker_keys/serverkey.pem -out /home/<USER>/docker_keys/servercert.pem -nodes -days 365
+			```
+            You will have to change the permissions for the serverkey.pem in order for the gvm user in the gsa container to access it.
+            ```
+            sudo chmod 660 /home/<USER>/docker_keys/serverkey.pem
+            ```
+            >[!WARNING] This is not secure, however, I have not found a better way. Using the default permissions (600), the gvm user inside the gsa container is unable to access the file once passed in.
+            
+
+        <details>
+        <summary>Example docker-compose.yml (Click):</summary>
```

Comment:
Neet, I have seen this done in others but we should use it more (That is HTML in MD docs)